client/Queries/Config/index.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+import gql from 'graphql-tag';
+
+export const GET_CONFIG = gql`
+  query {
+    client {
```

Comment:
Semantically, `about` doesn't feel like "config" - is this more designed to get "runtime" info?

Reply:
I think the config name is fine and fairly standard.